forensics/Copy-VmDigitalEvidenceWin.ps1

```diff
@@ -61,7 +67,8 @@ if ($bios) {
     # The Storage account also hosts an Azure file share to use as a temporary repository for calculating the snapshot's SHA-256 hash value.
     # The following doc shows a possible way to mount the Azure file share on Z:\ :
     # https://docs.microsoft.com/azure/storage/files/storage-how-to-use-files-windows
-
+    
+    # put here your code to map the file share. Note that the file share needs to be mapped by this script, otherwise the runbook worker'll miss the right access to the share.
```

Comment:
```suggestion
    # Put your code here to map the file share. Note that the file share needs to be mapped by this script, otherwise the runbook worker will not function due to missing access to the share.
```